verify/src/metadata_schema.json

```diff
@@ -166,7 +166,7 @@
       "type": "array",
       "minItems": 2,
       "maxItems": 2,
-      "items": [
+      "prefixItems": [
```

Comment:
Looks great, just linking the relevant jsonschema changelog entry here for posterity: https://json-schema.org/draft/2020-12/release-notes.html#changes-to-items-and-additionalitems